eng/config/PublishData.json

```diff
@@ -6,19 +6,19 @@
   "packages": {
     "default": {
       "Microsoft.CodeAnalysis": "arcade",
-      "Microsoft.CodeAnalysis.Common": "arcade",
+      "Microsoft.CodeAnalysis.Common": "vssdk",
```

Comment:
hmm this will stop publishing these via arcade I think.  Is that OK?  Feels like we still need to publish them to the dotnet feeds.

Reply:
All our packages are always published by Arcade to the dotnet-tools feed. This should additionally publish it to vssdk.

Reply:
Take EditorFeatures for example with is also configured as "vssdk". https://dev.azure.com/dnceng/public/_artifacts/feed/dotnet-tools/NuGet/Microsoft.CodeAnalysis.EditorFeatures/overview/4.5.0-1.22506.23